app/views/admin/npq/applications/_application_details.html.erb

```diff
@@ -3,113 +3,94 @@
     sl.row do |row|
       row.key(text: 'Application UUID')
       row.value(text: application.id || govuk_tag(text: "Missing", colour: "red"))
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'User UUID')
       row.value(text: application.user.id || govuk_tag(text: "Missing", colour: "red"))
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'User Identity UUID')
       row.value(text: application.user.id || '-')
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'Email')
       row.value(text: application.user.email)
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'Name')
       row.value(text: application.user.full_name)
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'TRN')
       row.value(text: application.teacher_reference_number || '-')
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'TRN validated')
       row.value(text: application.teacher_reference_number_verified.to_s)
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'Course')
       row.value(text: application.npq_course.name)
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'Lead provider')
       row.value(text: application.npq_lead_provider.name)
-      row.action
     end
 
     sl.row do |row|
       row.key(text: 'School URN')
       row.value(text: application.school_urn || '-')
       if application.school.present?
         row.action(text: "View School", href: admin_school_path(application.school))
```

Comment:
I recall hitting an issue previously where if one row had an action and the others didn't it'd look odd on the other rows because there'd be too few cells in the row. A total cellsize for the row of 3 but most columns only have two cells, leading to odd border issues. Are ou sure removing them from all the other rows won't cause that?